docs/develop/astraeus/getting-started.md

```
@@ -0,0 +1,413 @@
+
+- An environment capable of running Docker
+- Two accounts on the Toliman Testnet with access to their private keys and TEETH tokens
+- Two accounts on the Sepolia Testnet with access to their private keys and SepoliaETH tokens
+
+If you do not have these tokens, you can obtain them from the [Toliman Testnet Faucet](https://faucet.toliman.suave.flashbots.net/) and [Sepolia Testne Faucet](https://www.alchemy.com/faucets/ethereum-sepolia)
+
+### Steps
+
+1. **Copy the example environment file to create your own `.env` file:**
+
+   ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)